packages/ui-components/src/components/Map/Markers/MeasurePopup.js

```diff
@@ -9,9 +9,9 @@ import { getFormattedInfo } from '../utils';
 import { PopupMarker } from './PopupMarker';
 
 const buildHeaderText = (measureData, popupHeaderFormat) => {
-  const { organisationUnitCode, name } = measureData;
+  const { code, name } = measureData;
   const replacements = {
-    code: organisationUnitCode,
```

Comment:
Fix missed update on refactor of `organisationUnitCode`  to `code` to entity server